src/Features/SupportQueryString/BaseUrl.php

```diff
@@ -13,10 +14,13 @@ public function __construct(
         public $history = false,
         public $keep = false,
         public $except = null,
+        public $nullable = null,
     ) {}
 
     public function mount()
     {
+        $this->nullable = $this->determineNullability();
```

Comment:
I extracted that big blog of reflection conditional into a small method (makes readability much better)